packages/eas-cli/src/uploads.ts

```diff
@@ -31,6 +32,37 @@ export async function uploadWithPresignedPostAsync(
   file: string,
   presignedPost: PresignedPost,
   handleProgressEvent?: ProgressHandler
+): Promise<Response> {
+  return await promiseRetry(
+    async retry => {
+      let response: Response;
+      try {
+        response = await uploadWithPresignedPostInternalAsync(
+          file,
+          presignedPost,
+          handleProgressEvent
```

Comment:
Behavior to test: what happens on a retry when the progress potentially goes backwards (a retry will start over at 0%)?